src/commands/groups/util.ts

```diff
@@ -0,0 +1,16 @@
+export const outputFormat = {
+	table: [
+		{
+			header: 'ID',
+			property: 'id'
+		},
+		{
+			header: 'Name',
+			property: 'name'
+		},
+		{
+			header: 'Type',
```

Comment:
Why do we care about `type`?

Reply:
will remove this.  i don't fully understand the architecture of the users / groups, so i wasn't quite sure if this was useful.